### PR TITLE
Seabios support optiplex

### PR DESCRIPTION
**Pull request by filipleple**

### CODE REVIEW

**Comment by filipleple** [comment]
Here are the changes I needed to get dmidecode suite to execute as expected on OptiPlex 7010 with SeaBIOS. I assume all tests that just jump into OS immediately will work as well, so I'll mark them as `AUTOMATED`. I realize that the  `${SEABIOS_BOOT_DEVICE}=     2` hardcoding is a bit of a hack, will try to get it to work without it, but for the moment I think this is the most stable workaround to test this release.

```
==============================================================================
Dmidecode
==============================================================================

Checking if dmidecode is installed...

Package dmidecode is installed
DMI001.001 Verify the device serial number :: Check whether the DU... | PASS |
------------------------------------------------------------------------------
DMI002.001 Verify the firmware version :: Check whether the firmwa... | PASS |
------------------------------------------------------------------------------
DMI003.001 Verify the firmware product name :: Check whether the D... | PASS |
------------------------------------------------------------------------------
DMI004.001 Verify the firmware release date :: Check whether the f... | PASS |
------------------------------------------------------------------------------
DMI005.001 Verify the firmware manufacturer :: Check whether the f... | PASS |
------------------------------------------------------------------------------
DMI006.001 Verify the firmware vendor :: Check whether the firmwar... | PASS |
------------------------------------------------------------------------------
DMI007.001 Verify the firmware family :: Check whether the firmwar... | FAIL |
'# dmidecode 3.3
Getting SMBIOS data from sysfs.
SMBIOS 3.0.0 present.

Handle 0x0001, DMI type 1, 27 bytes
System Information
        Manufacturer: Dell Inc.
        Product Name: OptiPlex 9010
        Version: 1.0
        Serial Number: 123456789
        UUID: Not Settable
        Wake-up Type: Reserved
        SKU Number: Not Specified
        Family: Not Specified

Handle 0x0013, DMI type 32, 11 bytes
System Boot Information
        Status: No errors detected' does not contain 'Family: N/A'
------------------------------------------------------------------------------
DMI008.001 Verify the firmware type :: Check whether the firmware ... | PASS |
------------------------------------------------------------------------------
```

File: platform-configs/optiplex-7010-seabios.robot [hunk 1]
```diff
@@ -0,0 +1,12 @@
+*** Settings ***
+Resource    optiplex-7010.robot
+
+
+*** Variables ***
+${BIOS_LIB}=                        seabios
+${FW_STRING}=                       ESC
+# ${SEABIOS_BOOT_DEVICE}=    2
+${BOOT_MENU_KEY}=                   ${ESC}
+${DMIDECODE_FIRMWARE_VERSION}=      Dasharo (coreboot+SeaBIOS) v0.1.0
```
Comment (pietrushnic) [inline platform-configs/optiplex-7010-seabios.robot:10]: Are we sure about the hardcoding firmware version as well as the build date?

Comment (macpijan) [reply platform-configs/optiplex-7010-seabios.robot:10]: We can convert this discussion into issue and continue there @filipleple

Comment (macpijan) [reply platform-configs/optiplex-7010-seabios.robot:10]: > Are we sure about the hardcoding firmware version as well as the build date?

Maybe not, but that's how it used to be and we have it for all platforms. Need to be discussed separately and changed at the same time for all.

Version might be easier (take from file name?), release date perhaps not. Let's discuss in the issue.

Comment (pietrushnic) [reply platform-configs/optiplex-7010-seabios.robot:10]: Sure, let's move the discussion to the issue.

**Review by pietrushnic — APPROVED** [review]

File: platform-configs/optiplex-7010-seabios.robot (hunk 1 above)
Comment (filipleple) [reply platform-configs/optiplex-7010-seabios.robot:10]: https://github.com/Dasharo/open-source-firmware-validation/issues/632

**Pull request merged**